config/config.go

```diff
@@ -144,20 +144,34 @@ type Gen2Config struct {
 
 // VPCProviderConfig configures a specific instance of a VPC provider (e.g. GT/GC/Z)
 type VPCProviderConfig struct {
-	Enabled              bool   `toml:"vpc_enabled" envconfig:"VPC_ENABLED"`
+	Enabled bool `toml:"vpc_enabled" envconfig:"VPC_ENABLED"`
+
+	//valid values (gc|g2), if unspecified, GC will take precedence(if both are specified)
+	//during e2e test, user can specify its own preferred type during execution
+	VPCTypeEnabled       string `toml:"vpc_type_enabled" envconfig:"VPC_TYPE_ENABLED"`
```

Comment:
Who does populate the `VPCTypeEnabled` ? Driver ?
I think lib should automatically identify its based on `g2_api_key` or 'gc_api_key`